src/popup/locales/en.json

```
@@ -654,11 +665,35 @@
         "part1": "Superhero Wallet supports secure login with your mobile device authentication preferences.",
         "part2": "Your mobile device does not have secure authentication enabled currently. Would you like to enable it now?"
       },
-      "unsupportedDevice": "This feature is currently not supported by the Superhero Wallet browser extension. {'<'}br /{'>'} {'<'}br /{'>'} Secure login is available only to the users of Superhero Wallet mobile apps at the moment.",
       "walletLocked": "Wallet locked",
       "walletLockedSubtitle": "Tap on the screen to login!",
+      "enterPassword": "Enter password",
+      "toUnlock": "to unlock your wallet",
       "authFailed": "Authentication failed",
-      "authFailedSubtitle": "Tap on the screen to try again!"
+      "authFailedSubtitle": "Tap on the screen to try again!",
+      "setPassword": {
+        "title": "Set a password",
+        "text": "to securely access your wallet",
+        "text-2": "Choose a strong password and make sure you remember it or back it up safely.",
+        "textRestoredWallet-1": "Superhero Wallet has been updated to v.{0} introducing increased level of security.",
+        "textRestoredWallet-2": "You need to set a password to continue using your wallet. Chose a strong one and make sure you remember it or back it up safely.",
+        "passwordLabel": "Password",
+        "passwordPlaceholder": "Enter strong password",
+        "confirmLabel": "Confirm password",
+        "confirmPlaceholder": "Repeat the password you chose",
+        "confirm": "Continue to wallet",
+        "help": {
+          "title": "Set a password",
+          "text": "to securely access Superhero Wallet {'<'}br /{'>'} {'<'}br /{'>'} Superhero Wallet stores your seed phrase securely encrypted in the local storage. {'<'}br /{'>'} That’s why you need to set a password that will be used to unlock your wallet. {'<'}br /{'>'} {'<'}br /{'>'} If you forget the password the only way to access your wallet and funds will be to use the seed phrase (a mnemonic consisting of 12 words) in order to restore your accounts and set a new password. {'<'}br /{'>'} {'<'}br /{'>'} That is the reason why it is very important to remember your password and even more important to back up the seed phrase as soon as your wallet is created."
+        }
+      },
+      "login": {
+        "error": "The password you entered is incorrect.",
+        "placeholder": "Enter your password",
+        "label": "Password",
+        "unlock": "Unlock",
+        "forgot": "Forgot password"
```

```suggestion
        "forgot": "Forgot password?"
```
According to designs.